shell/utils/validators/formRules/index.ts

```diff
@@ -176,6 +176,17 @@ export default function(t: Translation, { key = 'Value' }: ValidationOptions): {
     return containers.map((container: any) => containerImage(container)).find((containerError: string) => containerError);
   };
 
+  const registryUrl = (privateRegistryURL: string) => {
+    const pattern = new RegExp('^([a-z\\-0-9]+:\\/\\/?)?' + // scheme (optional, https://, http://, file:/, admin:/)
```

Comment:
IF and only IF you are interested, you could add 1 line to the existing tests here:
https://github.com/rancher/dashboard/blob/master/shell/utils/validators/formRules/__tests__/index.test.ts#L1076

Something like:

```ts
    ['registryUrl', ['registry.io:5000'], []],
```

Reply:
Totally valid. I've added tests.